terraform/rds/rds.tf

```diff
@@ -0,0 +1,48 @@
+resource "aws_rds_cluster" "rdsclu" {
+    cluster_identifier      = var.cluster_name
+    engine                  = "aurora-mysql"
+    engine_version          = "5.7.mysql_aurora.2.07.2"
+    db_subnet_group_name    = var.db_subnet_group_name
+    vpc_security_group_ids  = [aws_security_group.rds-apps.id]
+    availability_zones      = var.availability_zones
+    database_name           = var.db_name
+    master_username         = "admin"
+    master_password         = "admin" # the password will be repalced when security manager set up
+    backup_retention_period = 1
+    deletion_protection     = false
+    apply_immediately       = true
+    skip_final_snapshot     = true
+}
+
+resource "aws_rds_cluster_instance" "cluster_instances" {
+    count                  = 1
+    identifier             = "aurora-cluster-instance"
+    cluster_identifier     = aws_rds_cluster.rdsclu.id
+    instance_class         = "db.r4.large" # this is the smallest instance can be use for the engine
+    engine                 = "aurora-mysql"
+    engine_version         = "5.7.mysql_aurora.2.07.2" #serverless is not supported for this version 
```

Comment:
can you try and see if Wordpress image connects to it with:
engine                  = "aurora"
engine_mode             = "serverless"

Reply:
I tried serverless, but got an error said serverless is not supported for this engine version, so instead I used the default mode, probably I can try use serverless with some other versions.

Reply:
that combination of `aurora` and `serverless` should do the trick

Reply:
I tried this, but not working, got an error `Error: error creating RDS cluster: InvalidParameterValue: The engine mode serverless you requested is currently unavailable.
        status code: 400, request id: 48d4b88e-40bc-4581-a801-38875e3203f0`